mlir/lib/Dialect/OpenMP/IR/OpenMPDialect.cpp

```diff
@@ -433,6 +433,47 @@ static void printScheduleClause(OpAsmPrinter &p, Operation *op,
     p << ", simd";
 }
 
+//===----------------------------------------------------------------------===//
+// Parser and printer for Order Clause
+//===----------------------------------------------------------------------===//
+
+// order ::= `order` `(` [order-modiﬁer ':'] concurrent `)`
```

Comment:
The grammar seems inaccurate. In the testcases it seems that the second value can be both `default` and `concurrent`. Can you check this once?